README.md

```diff
@@ -0,0 +1,108 @@
+########################################################
+#			Welcome to the Kokkos Documentation Workflow     #
+########################################################
+
+	Here, we summarize the steps for setting up the kokkos-core-wiki repo, and a
+branch-based workflow for updating documentation in the main Kokkos project.  The kokkos-core-wiki is the
+canonical source of Kokkos documentation "Truth".
+
+	CAVEAT: Creation of wholly new pages will require Review and Approval in some
+cases.  Check with your team lead, if you are uncertain.
+
+	STEP 1) Create your own fork of the kokkos-core-wiki repo by navigating to
+					https://github.com/kokkos/kokkos-core-wiki, and clicking "Fork" in the upper
+					right of the github page.
+
+	STEP 2) Clone your forked repo:
+
+					git clone git@github.com:ajpowelsnl/kokkos-core-wiki.git
+
+	STEP 3) Create a topic branch in your local fork of the kokkos-core-wiki:
+
+					git checkout -b demo/setup
+
+	STEP 4) Check your remotes:
+
+					[ajpowel@kokkos-dev-2 kokkos-core-wiki]$ git remote -v
+					origin	git@github.com:ajpowelsnl/kokkos-core-wiki.git (fetch)
+					origin	git@github.com:ajpowelsnl/kokkos-core-wiki.git (push)
+
+	STEP 5) Rename your (local) repo to disambiguate:
+
+					git remote rename origin ajpowelsnl
+
+	STEP 6) Add the remote kokkos-core-wiki repo (i.e., the repo you will create a PR on):
+
+					git remote add origin https://github.com/kokkos/kokkos-core-wiki.git
+
+	STEP 7) Add the Kokkos project Wiki from your fork (you will push changes to this repo to preview): 
+
+ 					git remote add my_wiki git@github.com:ajpowelsnl/kokkos.wiki.git
```

Comment:
For this to work, you have to set up the Wiki in the fork first, e.g., by just creating a dummy page.